Changed README (#13131)
demisto/content: Packs/Recorded_Future/README.md

> Note: This pack is deprecated and going to be EOL around November 2021, use Recorded Future v2 instead.

Reviewer: ```suggestion
Note: This pack is deprecated and will be declared End-of-Life around November 2021; use Recorded Future v2 pack instead.
```

> Note: This pack is deprecated and will be declared End-of-Life around November 2021; use Recorded Future v2 pack instead.

Reviewer 2: ```suggestion
Note: This pack is deprecated and will be declared End-of-Life around November 2021; use the Recorded Future v2 content pack instead.
```
